### ISSUE TITLE
How to create a basic neural network?

### ISSUE THREAD

**Issue by TheInvoker** [tensorflow/tfjs#512]
Hi,

Im new to this, and want to create a basic Neural network, that takes 3 inputs and 1 output. With 2 hidden layers of 3 nodes each. Then train it on some set of inputs, and then test it out with new values.

I saw some examples here

` 

    
    <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs"> </script>

    
    <script>
      // Notice there is no 'import' statement. 'tf' is available on the index-page
      // because of the script tag above.

      // Define a model for linear regression.
      const model = tf.sequential();
      model.add(tf.layers.dense({units: 1, inputShape: [1]}));

      // Prepare the model for training: Specify the loss and the optimizer.
      model.compile({loss: 'meanSquaredError', optimizer: 'sgd'});

      // Generate some synthetic data for training.
      const xs = tf.tensor2d([1, 2, 3, 4], [4, 1]);
      const ys = tf.tensor2d([1, 3, 5, 7], [4, 1]);

      // Train the model using the data.
      model.fit(xs, ys).then(() => {
        // Use the model to do inference on a data point the model hasn't seen before:
        // Open the browser devtools to see the output
        model.predict(tf.tensor2d([5], [1, 1])).print();
      });
    </script>
`

but can't figure out how to modify this for my situation.

**Comment by caisq** [comment]
@TheInvoker Thanks for the question. But stackoverflow.com might be more appropriate for how-to questions like this.
